api/v1alpha1/release_types.go

```diff
@@ -83,6 +96,11 @@ type ReleaseStatus struct {
 	// +optional
 	Target string `json:"target,omitempty"`
 
+	// Type indicates if this is a tenant release, managed release, or both
+	// +kubebuilder:validation:Pattern=^[a-z0-9]([-a-z0-9]*[a-z0-9])?$
+	// +optional
+	Type string `json:"type,omitempty"`
```

Comment:
Because I need a way to quickly determine if the release has only a tenant or only a managed pipeline. Otherwise I have to do like 5 lines of logic in a bunch of places